openapi/paths/storefront/cashier-deposit.yaml

```
@@ -0,0 +1,31 @@
+post:
+  x-products:
+    - Storefront
+  tags:
+    - Cashier
+  summary: Create a deposit
+  operationId: StorefrontPostCashierDeposit
+  x-sdk-operation-name: deposit
```

```suggestion
  x-sdk-operation-name: create
```

suggestion (non-blocking): 

I haven't seen any operation called `deposit`. I think the convention is to use `create` here.

---

Good catch. Maybe we should enforce a discrete amount of possible values in the linter?